packages/react-server-cli/src/index.js

```diff
@@ -1,7 +1,7 @@
 const fs = require('fs');
-const run = require("./run").default;
-const parseCliArgs = require("./parseCliArgs").default;
-const defaultOptions = require("./defaultOptions").default;
+const run = require("./run");
+const parseCliArgs = require("./parseCliArgs");
+const defaultOptions = require("./defaultOptions");
```

Comment:
Ooh, that's nice!  Didn't know about [`babel-plugin-add-module-exports`](https://www.npmjs.com/package/babel-plugin-add-module-exports).